File: Backend/src/main/java/by/gastrofest/dto/GastroSetDto.java
package by.gastrofest.dto;

public record GastroSetDto(
        Long id,
        String imageLink,
        String url,
        String theme,
        Integer weight,
        Boolean eatOutside,
        Boolean booking,
        Boolean delivery,
        String gastrofest,
        String participant,
        Boolean restaurant,
        java.util.List<String> mealsImages,
        java.util.List<String> mealsDescriptions
) {

}

File: Backend/src/main/java/by/gastrofest/dto/GastroSetDto.kt
package by.gastrofest.dto

data class GastroSetDto(
    var id: Long,
    var imageLink: String,
    val url: String,
    val theme: String,
    val weight: Int,
    val eatOutside: Boolean? = null,
    var booking: Boolean? = null,
    var delivery: Boolean? = null,
    val gastrofest: String,
    val participant: String,
    val restaurant: Boolean? = null,
    var mealsImages: List<String>,
    val mealsDescriptions: List<String>
)
